package.json

```diff
@@ -83,13 +86,12 @@
     "react": "^16.4.1",
     "react-addons-test-utils": "^15.6.2",
     "react-dom": "^16.4.1",
-    "react-styleguidist": "^6.5.3",
     "react-test-renderer": "^16.4.1",
     "semantic-release": "^12.4.1",
-    "style-loader": "^0.18.2",
     "styled-components": "^3.3.3",
     "travis-deploy-once": "^4.4.1",
-    "webpack": "^3.12.0"
+    "webpack": "^4.20.2",
```

Comment:
I thought this was supposed to be a big deal? Did it just go a lot smoother than we thought it would.

Reply:
webpack 4 is a requirement for `docz`. We're not doing a lot of fancy things with our webpack config in this repo so the upgrade was pretty straight forward. We'll want to test on ifixit/ifixit to make sure this didn't break anything though.